### DISCUSSION TITLE
Complete newbie getting started with first example

### DISCUSSION THREAD

**Discussion by johnhmccauley** [HomeSpan/HomeSpan#359] [Q&A]
I ran into a number of issues 

Esp32 using the latest Arduino IDE

I was unable to get the simple light bulb to compile using the latest version 1.5.1. There was an error with the LOG definition. Switched to the previous version of HomeSpan and did compile.

After running got the error in the serial window, fatal error stopped. Tried TwoSimpleLightBulbs and this did compile and the serial window said that it was working, no WiFi info, which I would expect.

Also the status light is not lit, I see a warning I need to set the pin.

 In file included from /Users/johnmccauley/Documents/Arduino/libraries/HomeSpan/src/HomeSpan.h:42:0,
                 from /Users/johnmccauley/Documents/Arduino/libraries/HomeSpan/src/HAP.h:32,
                 from /Users/johnmccauley/Documents/Arduino/libraries/HomeSpan/src/HAP.cpp:32:
/Users/johnmccauley/Documents/Arduino/libraries/HomeSpan/src/HAP.cpp: In member function 'void HAPClient::processRequest()':
/Users/johnmccauley/Documents/Arduino/libraries/HomeSpan/src/Settings.h:108:56: error: 'class HardwareSerial' has no member named 'print__VA_OPT__'
 #define LOG2(format,...) if(homeSpan.logLevel>1)Serial.print ##__VA_OPT__(f)(format __VA_OPT__(,) __VA_ARGS__)
                                                        ^
/Users/johnmccauley/Documents/Arduino/libraries/HomeSpan/src/HAP.cpp:190:5: note: in expansion of macro 'LOG2'
     LOG2("<<<< #### ");
     ^
Any help welcome.

**Comment by johnhmccauley** [comment]
I did figure out how to set the status pin. 

homeSpan.setStatusPin(2);

**Reply by HomeSpan** [reply]
Can you check to see you are running version 2.0.0 or greater of the Arduino-ESP32 board manager?

**Comment by johnhmccauley** [comment]
Running 1.0.6 esp32 by Expressif Systems. I will look for a later version. Thanks for your help.

**Reply by johnhmccauley** [reply]
Thanks, that fixed the problem. Was able to compile and add to Apple Home.
